packages/cms-cli/commands/remove.js

```diff
@@ -42,32 +75,30 @@ function configureRemoveCommand(program) {
       if (!(validateConfig() && (await validatePortal(command)))) {
         process.exit(1);
       }
-
-      const portalId = getPortalId(command);
-
-      trackCommandUsage(COMMAND_NAME, {}, portalId);
-
-      try {
-        await deleteFile(portalId, hsPath);
-        logger.log(`Deleted "${hsPath}" from portal ${portalId}`);
-      } catch (error) {
-        logger.error(`Deleting "${hsPath}" from portal ${portalId} failed`);
-        logApiErrorInstance(
-          error,
-          new ApiErrorContext({
-            portalId,
-            request: hsPath,
-          })
-        );
-      }
+      await action({ hsPath }, command);
     });
 
   addConfigOptions(program);
   addPortalOptions(program);
   addLoggerOptions(program);
   addHelpUsageTracking(program, COMMAND_NAME);
 }
+exports.command = `${COMMAND_NAME} <path>`;
```

Comment:
Why do we export command and describe?

Reply:
https://github.com/yargs/yargs/blob/master/docs/advanced.md#providing-a-command-module

Reply:
I see... then what is the exports.configureRemoveCommand for?

Reply:
For the existing Commander.js version of the CLI.

Reply:
Ahh.  I need to look into commander.js then.  Thanks for the info, making more sense